_layouts/midnight-train.html

```diff
@@ -358,6 +358,11 @@
 			font-weight: normal;
 			text-transform: uppercase
 		}
+
+		h1 svg
+		{
+			max-width: 500px;;
```

Comment:
This isn't like an "issue but can you remove the ;

Reply:
ya whoops